README.md

```diff
@@ -1,297 +1,236 @@
-# Learnosity SDK - PHP
+<p align="center"><img width="50%" height="50%" src="image-logo-graphic.png"></p>
+
+<h1 align="center">Learnosity SDK - PHP</h1>
+
+<p align="center">Everything you need to start building your app in Learnosity, with the PHP programming language.<br> 
+(Prefer another language? <a href="https://help.learnosity.com/hc/en-us/sections/360000194318-Server-side-development-SDKs">Click here</a>)<br>
+An official Learnosity open-source project.</p>
```

Comment:
This should be Markdown in here, not HTML.

Reply:
Being that this is the special "_GitHub-flavoured Markdown_", this is the way to do centred content with GitHub-flavoured Markdown. I think users will especially enjoy our unique presentation when rendered in a browser. :-)